packages/ibm-products-styles/src/components/FilterPanel/_filter-panel-checkbox.scss

```diff
@@ -20,16 +20,28 @@ $block-class: #{c4p-settings.$pkg-prefix}--filter-panel-checkbox;
 $label: #{c4p-settings.$pkg-prefix}--filter-panel-label;
 $carbon: #{c4p-settings.$carbon-prefix};
 
-.#{$block-class} .#{$carbon}--checkbox-label {
-  display: flex;
-  align-items: center;
+// UNDO Carbon's concept that a Checkbox is always a list item (":last-of-type").
+// This also allows the TruncatedList to calculate the correct height for expanding/collapsing the list.
+.#{$block-class}.#{$carbon}--form-item.#{$carbon}--checkbox-wrapper,
+.#{$block-class}.#{$carbon}--form-item.#{$carbon}--checkbox-wrapper:last-of-type {
+  margin-block-end: 0;
+}
+
+// Surround a checkbox's count with parentheses.
+.#{$block-class}
+  .#{c4p-settings.$pkg-prefix}--filter-panel-label__count::before {
+  content: '(';
```

Comment:
Not sure if parenthesis are considered decorative or non-decorative content? If they're non-decorative, then this will be an [a11y violation](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87).

Reply:
That's a good question; I don't have an answer for you other than:
1. Design confirmed that Checkboxes have counts with parens. I did bring up with Design that in accounting parens define negative numbers, and Design reconfirmed using parens.
2. Design confirmed that Groups and Accordions have counts without parens.
3. Original code appends parens via CSS as above.

I think they're purely decorative.

Reply:
Yes, I was going to say agree this is probably decorative. 🤔

Reply:
For more context —

JAWS would actually read this out[^1] if we included it as non-decorative. I think reading this as “6” would be preferable to “left paren 6 right paren” in this case.

[^1]: https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/